tools/build-all-docs.sh

```diff
@@ -47,22 +47,30 @@ function add_board {
 	done
 }
 
+function build_all_docs {
+    # Need to build one board to get things started.
+    BOARD=$1
+    shift
+    echo "Building docs for $BOARD"
+    pushd boards/$BOARD > /dev/null
+    make doc
+    popd > /dev/null
+    cp -r boards/$BOARD/target/thumbv7em-none-eabi/doc doc/rustdoc
+    ## Now can do all the rest.
+    for BOARD in $*
+    do
+        echo "Now building for $BOARD"
```

Comment:
This, or perhaps more so line 16, is probably redundant now, but that's just a tiny cosmetic thing